### PR TITLE
Updated Swagger Documentation reference to OpenAPI

### PR DESCRIPTION
**Pull request by mariodavid**

### CODE REVIEW

File: content/modules/rest/pages/api-docs.adoc [hunk 1]
```diff
@@ -19,18 +19,18 @@ The generic Swagger documentation contains descriptions for all endpoints the RE
...
-The Generic Swagger Documentation is available in `JSON` and `YAML` formats:
+The Generic OpenAPI Documentation is available in `JSON` and `YAML` formats:
 
-`/docs/swagger.yaml`:: YAML version of generic documentation.
-`/docs/swagger.json`:: JSON version of generic documentation.
+`/docs/openapi.yaml`:: YAML version of generic documentation.
+`/docs/openapi.json`:: JSON version of generic documentation.
 
-TIP: It is also available online at http://files.cuba-platform.com/swagger/7.2
+TIP: It is also available online at https://docs.jmix.io/openapi/1.0/
```
Comment (knstvk) [inline content/modules/rest/pages/api-docs.adoc]: Perhaps we should introduce an asciidoc variable (attribute) in antora-playbook.yml and antora-playbook.ci.yml, like `open_api_url = https://docs.jmix.io/openapi/1.0/`

Comment (mariodavid) [reply content/modules/rest/pages/api-docs.adoc]: ok, sure. Will do 👍

Comment (mariodavid) [reply content/modules/rest/pages/api-docs.adoc]: done: https://github.com/Haulmont/jmix-docs/pull/193/files#diff-9599f48ba8492b11280845131247843778be2a26b569f98c66a3bcdf3c8b62d6R27

**Comment by knstvk** [comment]
👍

**Pull request merged**